symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +3029,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
```

Comment:
This adds sympy's initialization time to symengine's initialization time. That should be avoided.